model/minsky.cc

```diff
@@ -1389,22 +1389,33 @@ namespace minsky
        {
          if (auto g=dynamic_cast<GodleyIcon*>(i->get()))
            {
+			 string newName;  
              if (type!=VariableType::flow)
                for (auto v: g->flowVars())
                  if (v->valueId()==name)
-                   throw error("flow variables in Godley tables cannot be converted to a different type");
+                   {
+					   newName=v->name()+"^{Flow}";
+                       VariableValues::iterator iv=variableValues.find(newName);
+                       if (iv==variableValues.end()) g->table.renameFlows(v->name(),newName);
+					   else throw error("flow variables in Godley tables cannot be converted to a different type");
```

Comment:
OK for now, but you can also create a loop, such that it tries ^{Flow}, ^{Flow1}, ... until it finds a name that doesn't clash.